notebooks/remote_functions/remote_function_usecases.ipynb

```diff
@@ -0,0 +1,1134 @@
+{
```

Comment:
<p>Explain this is in <code>pip</code> format and can include the version number. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/googleapis/python-bigquery-dataframes/pull/808/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Done.</p>